examples/tutorials/tutor.py

```diff
@@ -766,7 +768,7 @@ def run_code(self):
                         del values[name]
 
                 # Execute the current lab code:
-                exec module[2:] in values, values
+                six.exec_(module[2:] in values, values)
```

Comment:
I think we probably don't need `six` here: `exec(module[2:], values, values)` should work on both Python 2 and Python 3.